test/unit_tests/modules/util/standard-type-spec.js

```diff
@@ -7,5 +7,15 @@ var expect = chai.expect;
 var standardType = require( BASE_JS_PATH + 'modules/util/standard-type' );
 
 describe( 'standard-type', function() {
-  // TODO: Implement tests.
+  it( 'should include a standard JS data hook', function() {
+    expect( standardType.JS_HOOK ).to.equal( 'data-js-hook' );
+  } );
+
+  it( 'should include an filler function', function() {
+    expect( standardType.noopFunct() ).to.be.undefined;
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/2037/commits/2350084f2114e1c6a4dfcad4f0e25ab1d0f60cb8 .. though technically the test isn't checking if it's non operational, just that it returns undefined. It could be doing something, but I don't think it's one worth setting up a spy for, given its name and location.